src/schema/schema.ts

```diff
@@ -190,8 +190,14 @@ export function createZimbraSchema(
 						}))
 			},
 			Mutation: {
-				action: (_, variables) => {
+				action: (_, variables, context) => {
 					const { type, ...rest } = variables;
+					const { local } = context;
+
+					if (local && type === 'FolderAction') {
```

Comment:
please remove the `FolderAction` check from here and add it in the local client if necessary as we would have only one handler which caters to all the actions in the local client.